compose/config/config_schema_v3.7.json

```diff
@@ -597,6 +599,21 @@
           }
         }
       }
+    },
+    "device_requests": {
```

Comment:
leading schema definition is https://github.com/docker/cli/tree/master/cli/compose/schema/data 
see https://github.com/docker/compose/issues/7047
I'm looking into this schame duplication issue to be resolved asap

Reply:
That actually raises a very good point. Maybe the GPU allocation request should be based on [generic_resources](https://github.com/docker/cli/blob/master/cli/compose/schema/data/config_schema_v3.7.json#L439)?. Just sayin because of this: https://github.com/docker/cli/blob/master/cli/compose/loader/full-example.yml

But then that will require a translation to a [DeviceRequest](https://github.com/docker/docker-ce/blob/531930f3294c31db414f17f80fa8650d4ae66644/components/engine/api/types/container/host_config.go#L272)